client/src/pages/Home.js

```diff
@@ -59,7 +59,7 @@ export function Home() {
 						url={selectedIcon ? selectedIcon.video : ""}
 						location={selectedIcon ? selectedIcon.location : ""}
 						audiourl={selectedIcon ? selectedIcon.audio : ""}
-						description={selectedIcon ? selectedIcon.description: ""}
+						description={selectedIcon ? selectedIcon.description : ""}
```

Comment:
Can you remove the changes in Home.js? Doesn't matter that we don't link to MapSection